assets/src/dashboard/constants/stories.js

```diff
@@ -116,6 +116,7 @@ export const STORY_STATUS = {
   DRAFT: 'draft',
   FUTURE: 'future',
   PUBLISH: 'publish',
+  PRIVATE: 'private',
```

Comment:
Should also add this to `ALL` above, no?

Reply:
Nope, like above, private stores are not part of all, but a separate group. Now this could change and it would be easy to add.